pkgs/applications/version-management/gitlab/default.nix

```diff
@@ -17,7 +17,7 @@ let
 
   rubyEnv = bundlerEnv rec {
     name = "gitlab-env-${version}";
-    ruby = ruby_3_0;
```

Comment:
But afaik, GitLab isn't officially compatible with 3.2?

- 3.2: https://gitlab.com/gitlab-org/gitlab-development-kit/-/issues/1731
- 3.1 (which works): https://gitlab.com/groups/gitlab-org/-/epics/10034

Reply:
Did I upgrade it to 3.2 by mistake? Then, we should go back to 3.1, sorry for that.